ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -672,29 +651,23 @@ private void addToEndOfPool(ReplicaId replicaId) {
    * @param delta the value to add to the counter.
    */
   private void modifyReplicasInPoolOrInFlightCount(ReplicaType replicaType, int delta) {
-    if (replicaType == ReplicaType.CLOUD_BACKED) {
-      cloudReplicaInPoolOrFlightCount += delta;
-    } else {
-      diskReplicaInPoolOrFlightCount += delta;
-    }
+    replicaInPoolOrFlightCount += delta;
```

Comment:
we don't need ReplicaType in this method.

Reply:
Done in
212d358